loader/loader.go

```diff
@@ -0,0 +1,57 @@
+package loader
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+)
+
+type StandData struct {
+	StandID        string  `json:"standId"`
+	Stand         string  `json:"stand"`
+	Power       string  `json:"power"`
+	Speed       string  `json:"speed"`
+	Range       string  `json:"range"`
+	Persistence       string  `json:"persistence"`
+	Precision       string  `json:"precision"`
+	Development       string  `json:"development"`
+
+}
+
+func LoadData(r io.Reader) *[]*StandData {
```

Comment:
Since there are a lot of places where this function could generate an error, then the function should return the data plus an error `func LoadData(r io.Reader) (*[]*StandData, error) { ... }`